src/test/java/com/wikia/webdriver/testcases/mercurytests/SEOTests.java

```diff
@@ -16,20 +22,24 @@
  */
 public class SEOTests extends NewTestTemplate {
 
+  private static final List<String> ROBOTS_TAG_ATTRIBUTES_INDEX =
+      Arrays.asList("index", "follow");
+  private static final List<String> ROBOTS_TAG_ATTRIBUTES_NOINDEX =
+      Arrays.asList("noindex", "follow");
   private static final String MUPPET_MAIN_PAGE = "Muppet_Wiki";
 
   @BeforeMethod(alwaysRun = true)
   public void prepareTest() {
     driver.manage().timeouts().pageLoadTimeout(100, TimeUnit.SECONDS);
-    wikiURL = urlBuilder.getUrlForWiki("muppet");
   }
 
   // SEOT01
   @RelatedIssue(issueID = "HG-671")
   @Test(groups = {"MercurySEOTest_001", "MercurySEOTests", "Mercury"})
   public void MercurySEOTest_001_MetaTags_CanonicalLink() {
+    wikiURL = urlBuilder.getUrlForWiki("muppet");
     NavigationSideComponentObject leftNav = new NavigationSideComponentObject(driver);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "leftNav" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1481)